modules/commands/list.json

```diff
@@ -520,6 +520,24 @@
       }
     ]
   },
+  {
+    "name": "tab",
+    "aliases": [
+      "tablist"
+    ],
+    "title": "Tablist",
+    "description": "LuckPerms is only able to send information to formatting plugins so that they display them properly. LuckPerms does not put the prefix in front of your name on the tablist, you need a tab formatting plugin to do it, and you need Vault. Same goes for the chat! If the information appears correctly when you run `/lp user <user> info`, LuckPerms is doing its job!",
+    "fields": [
+      {
+        "key": "Prefix isn't showing?",
+        "value": "https://github.com/lucko/LuckPerms/wiki/FAQ#why-are-prefixessuffixes-not-working"
+      },
+      {
+        "key": "How can I use TAB to format my tablist?",

```

Comment:
Assuming that a person will use TAB is probably not a great idea for this embed when there are alternatives that should be suggested